cmd/podman/validate/args.go

```diff
@@ -86,6 +86,11 @@ func CheckAllLatestAndIDFile(c *cobra.Command, args []string, ignoreArgLen bool,
 		specifiedIDFile = true
 	}
 
+	specifiedFilter := c.Flag("filter").Changed
+	if specifiedFilter {
+		return nil
+	}
```

Comment:
I think if you're going to check for filter flag changes here, you should also move the `no arguments needed with --filter`, provided you checked that this won't conflict with any other commands that use filter.